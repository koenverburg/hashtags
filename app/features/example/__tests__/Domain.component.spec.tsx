/* tslint:disable no-implicit-dependencies */
import React from 'react'
import { shallow } from 'enzyme'
import { Provider } from 'mobx-react'
import { Domain } from '..'
import { DomainStore } from '../Domain.store'

describe('feature - <Domain />', () => {
  let store: DomainStore

  beforeAll(() => {
    store = new DomainStore()
  })

  it('renders', () => {
    const feature = shallow(
      (
        <Provider store={store}>
          <Domain />
        </Provider>
      ),
    )

    expect(feature).toMatchSnapshot()
  })
})
