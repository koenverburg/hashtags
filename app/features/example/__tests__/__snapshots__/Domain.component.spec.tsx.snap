// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`feature - <Domain /> renders 1`] = `
<ContextProvider
  value={
    Object {
      "store": DomainStore {
        "entities": undefined,
        "error": undefined,
        "status": "idle",
      },
    }
  }
>
  <inject-with-domain(Domain) />
</ContextProvider>
`;
